content/tutorial/02-advanced-svelte/02-transitions/08-key-blocks/README.md

```diff
@@ -2,9 +2,9 @@
 title: Key blocks
 ---
 
-Key blocks destroy and recreate their contents when the value of an expression changes. This is useful if you want an element to play its transition whenever a value changes instead of only when the element enters or leaves the DOM.
+Key blocks destroy and recreate their contents when the value of the following expression (in the example: `i`) changes. This is useful if you want an element to play its transition whenever a value changes instead of only when the element enters or leaves the DOM.
```

Comment:
I find the previous sentence to be more readable, please revert this change. This is a general introductionary sentence to the key blocks.